client/state/jetpack-connect/test/selectors.js

```diff
@@ -371,6 +372,56 @@ describe( 'selectors', () => {
 		} );
 	} );
 
+	describe( '#isSiteBlacklistedError', () => {
+		const stateHasBeenBlacklistedError = {
+			jetpackConnect: {
+				jetpackConnectAuthorize: {
+					authorizeError: {
+						error: 'site_blacklisted',
+					},
+				},
+			},
+		};
+
+		const stateHasNoError = {
+			jetpackConnect: {
+				jetpackConnectAuthorize: {
+					authorizeError: false,
+				},
+			},
+		};
+
+		const stateHasOtherError = {
+			jetpackConnect: {
+				jetpackConnectAuthorize: {
+					authorizeError: {
+						error: 'already_connected',
+					},
+				},
+			},
+		};
```

Comment:
Personally, I think I'd move each of these states into their corresponding `test()`s, but that's obviously a matter of taste.

Reply:
👍 done in https://github.com/Automattic/wp-calypso/pull/29983/commits/46e6d3af82ef4520f8f680009a691f4a164357b7